### ISSUE TITLE
too low accuracy

### ISSUE THREAD

**Issue by agnieszkau** [keras-team/keras#14729]
df = pd.read_csv('data.csv', index_col=0)
df = df.drop('Unnamed: 32', axis=1)

X = df.iloc[:,  2:].values
y = df.iloc[:,  1].values

from sklearn.preprocessing import LabelEncoder
labelencoder_X_1 = LabelEncoder()
y = labelencoder_X_1.fit_transform(y)

from sklearn.model_selection import train_test_split
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size = 0.3, random_state = 0)

from sklearn.preprocessing import StandardScaler
sc = StandardScaler()
X_train = sc.fit_transform(X_train)
X_test = sc.transform(X_test)

from tensorflow.keras.layers import Dense

model3 = Sequential()

model3.add(Flatten())
model3.add(layers.Dense(16, activation = "relu", kernel_initializer="uniform", input_dim =30))
model3.add(Dropout(0.1))
model3.add(layers.Dense(16, activation="relu", kernel_initializer="uniform"))
model3.add(Dropout(0.1))
model3.add(layers.Dense(1, activation="sigmoid", kernel_initializer="uniform"))

model3.compile(optimizer=keras.optimizers.Adam(),
              loss= keras.losses.categorical_crossentropy,
              metrics= ['accuracy'])

model3.fit(X_train, 
           y_train, 
           batch_size=100, 
           epochs=100)

Results:
![obraz](https://user-images.githubusercontent.com/86022292/122283763-2a77d700-ceed-11eb-9530-495873f1112c.png)

What to do to increase the accuracy?

**Comment by pedro-r-marques** [comment]
@agnieszkau This is most likely a question for stackoverflow and not something that should be a github issue.
Having a ```categorical_crossentropy``` loss in a model which has a 1 neuron output is most likely a mistake. You most likely meant to use ```binary_crossentropy```.

**Comment by jvishnuvardhan** [comment]
@agnieszkau Please check more info about usage of losses https://keras.io/api/losses/probabilistic_losses/.
If you still think this is a bug/performance issue, please share the data and standalone code to reproduce the issue.

If this was already resolved, please close the issue. Thanks!

**Comment by jvishnuvardhan** [comment]
Closing this issue as there is no recent activity and mostly the comment would have resolved the above issue. Please feel free to reopen if I am mistaken. Thanks!
